pkg/crane/crane_test.go

```diff
@@ -148,6 +147,33 @@ func TestCraneRegistry(t *testing.T) {
 	if err := compare.Layers(pulledLayer, layer); err != nil {
 		t.Fatal(err)
 	}
+
+	// Delete the non existing image
+	err = crane.Delete("honk-image")
```

Comment:
style nit: If a method only returns an error, you can scope that error to an if block that checks that error:

```go
if err := crane.Delete("honk-image"); err == nil {
  t.Fatal(...)
}
```

This has the benefit of not polluting outer scope, and can help readability since the reader can know the call and check won't have any effect on the rest of the code. It's also consistent with other code in the file, like `compare.Layers` above on line 147.

Reply:
thanks for the review